src/Microsoft.Azure.WebJobs.Extensions.Redis/PollingTriggerBase/RedisPollingTriggerBaseListener.cs

```diff
@@ -45,6 +49,7 @@ public RedisPollingTriggerBaseListener(string name, IConnectionMultiplexer multi
         /// </summary>
         public virtual Task StartAsync(CancellationToken cancellationToken)
         {
+            multiplexer = RedisExtensionConfigProvider.GetOrCreateConnectionMultiplexer(configuration, connectionStringSetting, name);
```

Comment:
Can the multiplexer initialization be a part of the constructor?

Reply:
The multiplexer initialization is going to become async, and we can't have async code in constructors